### ISSUE TITLE
Geometry_Engine: IsVertical methods

### ISSUE THREAD

**Issue by rwemay** [BHoM/BHoM_Engine#1199]
#### Description:

Add is vertical methods for Element1D and possibly Element2D(?) Checking whether a line or a planar surface based object is vertical is something that recurs in many disciplines, and there are already some methods (e.g. structural bar). Could also come in handy for the 'tilt' methods below. https://github.com/BHoM/BHoM_Engine/issues/1197

**Comment by pawelbaran** [comment]
Sounds good - changed the header to Common_Engine as it should work on IElements I think.

**Comment by FraserGreenroyd** [comment]
@pawelbaran @LMarkowski is this likely to be actioned next week? If not we should push this to 3.1?

Also is `Common_Engine` the right engine to place this, I thought all common engine methods were migrating out?

**Comment by pawelbaran** [comment]
Yeah, `IElements` are meant to migrate to the Geometry universe altogether - #1316 should go first I think. I will update the Milestone and title.

**Comment by kThorsager** [comment]
Can note that this is partly handled in #1402 by moving some functionality from Structure_Engine to Geometry. Will however not fix IElement things before #1316

**Comment by pawelbaran** [comment]
@kThorsager is that issue still relevant?

**Comment by kThorsager** [comment]
I haven't fixed it, so would suppose so.

But `IElement` solutions should be in Spatial_Engine, but unsure if methods for `IGeometries` are requested too
